app/export.go

```diff
@@ -161,7 +158,12 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 	counter := int16(0)
 
 	for ; iter.Valid(); iter.Next() {
-		addr := sdk.ValAddress(iter.Key()[1:])
+		key := iter.Key()
+		if len(key) <= 2 {
```

Comment:
```suggestion
		if len(key) < 3 {
```
I think this is clearer